pipelines/incubator/manual-pipeline-runs/java-microprofile-deploy-task.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: ${collection-name}-deploy-task
+spec:
+  inputs:
+    resources:
+      - name: git-source
+        type: git
+      - name: docker-image
+        type: image
+    params:
+      - name: repository-name
+        default: java-microprofile
```

Comment:
Could you update this also please?

Reply:
Ok, so that means above change would be again with the ${collection-name} variable subsitution ? 
Let me make that change.

Reply:
Done, Please verify : @kvijai82